app/parse.py

```diff
@@ -1,5 +1,9 @@
 from dataclasses import dataclass
 from enum import Enum
+import requests
+from bs4 import BeautifulSoup
+
+BASE_URL = "https://mate.academy/"
```

Comment:
place constant(s) into class/function or program start-point to not create redundant imports at the unexpected program places

Reply:
Change place, put constant BASE_URL in to Course class.